Sources/Result.swift

```diff
@@ -1,6 +1,7 @@
 /// `Result` is used to deliver results mapped in the `Bar`.
 public enum Result<M: Parseable> {
     case model(M?)
+    case ModelArray([M])
```

Comment:
🖕🏽Should be lower case: Swift 3

Also I would prefer to use `case models([M])`.